tools/tplg_parser/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+cmake_minimum_required(VERSION 3.10)
+
+project(SOF_TPLG_PARSER C)
+
+set(sof_source_directory "${PROJECT_SOURCE_DIR}/../..")
+
+add_library(sof_parser_inc INTERFACE)
```

Comment:
This target is not needed as you use it only to get `${CMAKE_CURRENT_SOURCE_DIR}/include`. I think you got it from the main sof project, but there it's needed because include dirs come from many sources, here you have just 1 dir and it's overcomplicated apporach.